SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -136,6 +137,12 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 
 		if (EntityId != SpatialConstants::INVALID_ENTITY_ID)
 		{
+			// If the entity already exists, make sure we have spatial authority before we replicate with Offloading, because we pretend to have local authority
+			if (USpatialStatics::IsSpatialOffloadingEnabled() && !NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::POSITION_COMPONENT_ID))
```

Comment:
I would like to use something else than the position component, but I don't think we have a nice function for checking authority over the actually relevant components.